x/ecocredit/server/invariants_test.go

```diff
@@ -27,16 +28,17 @@ import (
 )
 
 type baseSuite struct {
-	t            *testing.T
-	db           ormdb.ModuleDB
-	stateStore   api.StateStore
-	ctx          context.Context
-	k            coreserver.Keeper
-	ctrl         *gomock.Controller
-	bankKeeper   *mocks.MockBankKeeper
-	paramsKeeper *mocks.MockParamKeeper
-	storeKey     *sdk.KVStoreKey
-	sdkCtx       sdk.Context
+	t             *testing.T
+	db            ormdb.ModuleDB
+	stateStore    api.StateStore
+	ctx           context.Context
+	k             coreserver.Keeper
+	ctrl          *gomock.Controller
+	bankKeeper    *mocks.MockBankKeeper
+	paramsKeeper  *mocks.MockParamKeeper
+	moduleAddress sdk.AccAddress
```

Comment:
does this need to be a baseSuite field? its never used outside of keeper instantiation

Reply:
https://github.com/regen-network/regen-ledger/pull/1086/commits/86c6b6de1c0fe529a3a2537f091ba878564692fc